package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@yfi/yearn-web-lib",
```

Comment:
After some talk, it will be `@yearn/yearn-web-lib` 😄

Reply:
Changed with https://github.com/Major-Eth/yearn-web-lib/pull/8/commits/fabe51e1ad7d1dcf65d0cf2a8b581dda07f73c59